src/Microsoft.Health.Dicom.Core/Features/Store/StoreValidationResultBuilder.cs

```diff
@@ -59,6 +63,15 @@ public void Add(ValidationWarnings warningCode, QueryTag queryTag = null)
         }
     }
 
+    /// <summary>
+    /// Adds a tag to a list representing invalid Dicom items.
+    /// </summary>
+    /// <param name="tag">Invalid item's tag to add.</param>
+    public void Add(DicomTag tag)
```

Comment:
I don't think it's clear from the name of the method and the param what is happening here - how about changing the name of this method to `AddInvalidTag`?